### ISSUE TITLE
NTR, pulmo: Pulmonary Bleb

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5580]
**Preferred term label:**
Pulmonary Bleb

**Synonyms**

**Definition (free text, please give PubMed ID)**
A bleb is a small gas-containing space within the visceral pleura or in the subpleural lung, not larger than 1 cm in diameter. CT findings show a bleb as a thin-walled cystic air space contiguous with the pleura.

PMID: 18195376, PMID: 12934786

**Parent term (use hpo.jax.org/app)**
Emphysema, HP:0002097

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Term updated!
